spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -310,7 +328,7 @@ def get_decision_product(resp)
           hints: true,
         },
         attention_with_barcode: false,
-        doc_type_supported: true,
+        doc_type_supported: true, # todo: check failure document need update?
```

Comment:
Have we resolved this todo?

Reply:
@night-jellyfish , just did, mainly some of the tests file missing "DocIssuerType" field.